rpm/gst-plugins-good.spec

```diff
@@ -5,7 +5,7 @@
 %global _vpath_builddir subprojects/gst-plugins-good/_build
 
 Name: 		%{gstreamer}%{majorminor}-plugins-good
-Version: 	1.22.4
+Version: 	1.20.10
```

Comment:
I think there is a typo in the version number here....

Reply:
So it seems, doesn't matter much though since in OBS build the version is taken from the tag.